pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "dbt-semantic-interfaces"
-version = "0.1.0rc1"
+version = "0.1.0rc2"
```

Comment:
I think we should actually stop bumping the version as part of a feature PR. It's caused some discrepancies in the past. Releasing from a feature branch, releasing before the PR has been merged, etc. We should probably move to bumping the version as part of the release process, separate from any feature branch.